git/hooks/pre-commit

```diff
@@ -0,0 +1,13 @@
+#!/bin/sh
+files=`git diff --cached --name-status`
```

Comment:
I think this needs a `set -e` , or it will amend the wrong commit.  Both scripts should probably have it.